src/main/java/com/google/step/youtube/YouTubeInfoScraper.java

```diff
@@ -103,6 +105,21 @@ public YouTubeInfoScraper() {
         return Optional.of(response.getItems());
     }
 
+    /**
+     * @param videoIds List of ids of youtube videos.
+     * @return an optional list of Videos. The optional will be empty if id is
+     *         invalid or no items were found.
+     */
+    public Optional<List<Video>> scrapeVideoInformation(List<String> videoIds) throws IOException {
```

Comment:
Perhaps rename this to `scrapeVideosFromIds` or `scrapeVideos` since you are returning actual Video types and not just general information.

Reply:
Change description to be more clear on the return.